script.js

```diff
@@ -1,12 +1,122 @@
+                                 </ul>
+                             </div>
+                         </div>
+                         <div class="card-body">
+                              <p>${a.mainText}</p>
+                             <div>
+                                 <ul class="buttons-work-container">
+                                     <li class="btn-work-style">${a.technologies[0]}</li>
+                                     <li class="btn-work-style">${a.technologies[1]}</li>
+                                    <li class="btn-work-style">${a.technologies[2]}</li>
+                                 </ul>
+                             </div>
+                         </div>
+                         <div class="card-button">
+                             <button onclick="getProject(${a.id})" class="button-work-project">SEE PROJECT</button>
+                         </div>
+                     </div>
+                 </div>
+             </div>`;
+          
+          document.getElementById("outputMain").innerHTML = outputMain;   
+        });
+
+        document.getElementById("1").style.order = 1;
+        document.getElementById("3").style.order = 1;
+        
+ }
+)}
+
+/* Dynamic Popup Project Functions */
+function getProject(a) {
+  projectOn();
+fetch("projects.json")
+.then((res) => res.json())
+.then((data) => {
+ output = `
+ <div class="project_overlay">
+  <div class="project_structure_overlay">
+      <div> 
+          <button onclick="projectOff()" class="project_overlay_btn_close"> X </button>
+      </div>
+      <div class="project_heading_overlay">
+              <h2 class="title_overlay"> ${data[a].name} </h2>
+              <ul class="project_body_title_list">
+                  <li class="title_list_main_overlay">${data[a].projectClient}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[0]}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[1]}</li>
+              </ul>
+      </div>
+      <div> <img class="image_ovlerlay" src=${data[a].image} alt="snapshoot"> </div>
+      <div class="project_body_overlay">
+          <div class="text_group_overlay">
+              <p>${data[a].mainText}</p>
+          </div>
+          <div class="btn_group_overlay">
+          <ul class="project_tech_overlay">
+                  <li>${data[a].technologies[0]}</li>
+                  <li>${data[a].technologies[1]}</li>
+                  <li>${data[a].technologies[2]}</li>
+              </ul>
+              <hr class="hr_style">
+              <div class="btn_flex_overlay">
+                <button class="overlay_btn"> <a href="${data[a].liveVersionLink}"> See Live <img src="./Assets/Icon_Export.png" > </a> </button>
+                <button class="overlay_btn"> <a href="${data[a].liveVersionLink}"> See Source <img src="./Assets/github_in_button.png" > </a> </button>
```

Comment:
As requested by the previous reviewer, kindly check and make sure that the buttons of the `see live` and `see source` should conform with that of the figma design at all screen widths. Please check the font-size, paddings and margins etc. :wink: 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-17 18-58-59](https://user-images.githubusercontent.com/65233860/202536061-587df4dd-7330-43ea-b3c3-0eead680fac1.png)  |  ![Screenshot from 2022-11-17 19-05-44](https://user-images.githubusercontent.com/65233860/202535976-5f8eee35-f153-455c-b28b-f7e30746c074.png)